app/src/main/java/com/vestrel00/daggerbutterknifemvp/App.java

```diff
@@ -34,26 +30,15 @@
  * so that we have to option to inherit from something else later on if needed
  * (e.g. if we need to override MultidexApplication).
  */
-public class App extends Application implements HasActivityInjector {
-
-    @Inject
-    DispatchingAndroidInjector<Activity> activityInjector;
+public class App extends DaggerApplication {
```

Comment:
I created setup that uses *dagger.android.support* with AppCompatActivity and support Fragments. Take a look at #49.